ignite/metrics/__init__.py

```diff
@@ -7,8 +7,7 @@
 from ignite.metrics.mean_squared_error import MeanSquaredError
 from ignite.metrics.metric import Metric
 from ignite.metrics.epoch_metric import EpochMetric
-from ignite.metrics.precision import Precision
-from ignite.metrics.recall import Recall
+from ignite.metrics.precision_recall import Precision, Recall
```

Comment:
I'm not sure about to unify the files, this is a breaking change. The common class `_Base*` class we can store in `precision.py` and import then from `recall.py`

Reply:
Done!